lib/features/get_started/get_started_view.dart

```diff
@@ -110,7 +111,9 @@ class GetStartedView extends StatelessWidget {
                     Align(
                       alignment: Alignment.center,
                       child: ElevatedButton(
-                        onPressed: () {},
+                        onPressed: () {
+                          Navigator.of(context).push(MaterialPageRoute(builder: (context) => const AuthScreen()));
```

Comment:
use pushReplacement because we don't need to return on onboarding page after it's done

Reply:
done